tests/integration/records/test_records_receivers.py

```diff
@@ -36,17 +37,30 @@
     User,
     UserIdentity,
 )
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_pidstore.models import RecordIdentifier
 
 from inspirehep.modules.migrator.tasks import migrate_and_insert_record
 from inspirehep.modules.records.api import InspireRecord
 from inspirehep.modules.records.exceptions import MissingInspireRecord
+from inspirehep.modules.records.tasks import batch_reindex
 from inspirehep.modules.search import LiteratureSearch
 from inspirehep.utils.record import get_title
+from inspirehep.utils.record_getter import get_es_record
 
 from utils import _delete_record
 from factories.db.invenio_records import TestRecordMetadata
 
 
+def remove_record_from_db(record):
+    pid = PersistentIdentifier.get('lit', record['control_number'])
+    recid = RecordIdentifier.query.filter_by(recid=record['control_number']).one()
+    db.session.delete(record.model)
+    db.session.delete(pid)
+    db.session.delete(recid)
+    db.session.commit()
+
+
```

Comment:
https://github.com/inspirehep/inspire-next/blob/bf85efbbdb57d94e987d1714e383420a82510c19/tests/integration/helpers/utils.py#L39

Reply:
yeah, it was not here before :P